pandas-stubs/core/dtypes/missing.pyi

```diff
@@ -23,9 +26,11 @@ isneginf_scalar = ...
 @overload
 def isna(obj: DataFrame) -> DataFrame: ...
 @overload
-def isna(obj: Series) -> Series[bool]: ...
+def isna(obj: Series[Any]) -> Series[bool]: ...
 @overload
-def isna(obj: Index | list | ArrayLike) -> npt.NDArray[np.bool_]: ...
+def isna(
+    obj: Index[Any] | list[Any] | ExtensionArray | np.ndarray[Any, Any]
```

Comment:
Given that you redefined `ArrayLike` in `_typing.pyi`, couldn't you just use `ArrayLike` here?

Reply:
Done